lib/yelp/error.rb

```diff
@@ -47,13 +47,14 @@ def self.check_for_error(response)
 
     class Base < StandardError
       def initialize(msg,error=nil)
+        super(msg)
       end
     end
 
     class AlreadyConfigured < Base
       def initialize(msg = 'Gem cannot be reconfigured.  Initialize a new ' +
           'instance of Yelp::Client.', error=nil)
-        super
+      super
```

Comment:
fixed

On Fri, Jul 1, 2016 at 1:31 PM, Tomer Elmalem <notifications@github.com>
wrote:

> In lib/yelp/error.rb
> <https://github.com/Yelp/yelp-ruby/pull/53#discussion_r69331881>:
>
> >        end
> >      end
> >
> >      class AlreadyConfigured < Base
> >        def initialize(msg = 'Gem cannot be reconfigured.  Initialize a new ' +
> >            'instance of Yelp::Client.', error=nil)
> > -        super
> > +      super
>
> Could you indent this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Yelp/yelp-ruby/pull/53/files/bf5dc6e980a0fce97928de8208d671e6293f4c21#r69331881>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPH5iy9gcb5PqWzkd1Qu5OVTENPPSXDks5qRU7xgaJpZM4JDOBL>
> .
>